### ISSUE TITLE
Plex Request Status

### ISSUE THREAD

**Issue by Gamer239** [baldr-io/baldr.io#4]
We should have a status on requests. If its per user that's fine. If its public we need a general api for the website and the website should be super general like (requested, in-progress, done) for something. We don't want to say something that conflicts with general law throughout the world but it would be nice to inform people on the status even if its an over arching progress bar for all movies/ tv shows.

**Comment by Gamer239** [comment]
We need to make an api but it would be nice if we could give users stats on total movies, tv shows and what was updated recently. Plexwatchweb does this but we dont want to fully use users habits as this is a bit to personal.

**Comment by KPDerosh** [comment]
Guys get something that I can call through Ajax and is json I can put it on the front page. Like I call baldr.io/serverstatus and its just a php script that echos json about the server.

**Comment by Gamer239** [comment]
Yeah this plex watch status is something that Jake and I need to talk about
and provide you with. Don't worry about this until we do. I mentioned it
hopefully but at this point we don't have an api for you to access it
easily.

**Comment by Kelwing** [comment]
The API engine is done, we just need LUA scripts to power the endpoints.  Any of us can work on them.  I will be adding examples to the repository ASAP.
